P0009/P0009.md

```diff
@@ -2334,9 +2334,9 @@ constexpr mapping(const extents_type& e, const array<OtherSizeType, @_rank\__@>&
 
 [1]{.pnum} *Constraints:*
 
-   * [1.1]{.pnum} `is_convertible_v<const remove_const_t<OtherSizeType>&, size_type>` is `true`,
+   * [1.1]{.pnum} `is_convertible_v<const OtherSizeType&, size_type>` is `true`,
```

Comment:
I actually didn't like this change, but whatever, I guess it's Library convention.